addon/-legacy-private/system/relationships/state/belongs-to.js

```diff
@@ -231,17 +231,17 @@ export default class BelongsToRelationship extends Relationship {
     Other calls must conform to the typical expectations, for instance, sync relationships
     expect that their data is already loaded.
    */
-  getData(isForcedReload = false) {
+  getData(options = {}) {
```

Comment:
let's leave `isForcedReload` as the second argument here, as it should not be leaked to end users.

Reply:
options does not need a defaultValue